exporter/kinesisexporter/exporter.go

```diff
@@ -0,0 +1,220 @@
+package kinesisexporter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kinesis"
+	"github.com/aws/aws-sdk-go-v2/service/kinesis/types"
+	"github.com/thomaspoignant/go-feature-flag/exporter"
+	"github.com/thomaspoignant/go-feature-flag/utils/fflog"
+)
+
+const (
+	formatJSON = "json"
+	Mb         = 1024 * 1024
+)
+
+var DefaultPartitionKey = func(event exporter.FeatureEvent) string {
```

Comment:
Proposal to avoid any linter issue
```suggestion
var DefaultPartitionKey = func(_ exporter.FeatureEvent) string {
```